common/api/summary/appui-react.exports.csv

```diff
@@ -400,6 +407,7 @@ alpha;SheetNavigationAid
 alpha;SheetNavigationAidControl 
 alpha;SheetNavigationProps 
 alpha;SheetsModalFrontstage 
+public;ShowComponentOptions
```

Comment:
Similar to component props, can we not export Args/Options type for easier backwards compatibility?

Reply:
Yep, fixed.